diff --git a/node_modules/react-native/scripts/packager.sh b/node_modules/react-native/scripts/packager.sh
index 6ad9250..12c56c1 100755
--- a/node_modules/react-native/scripts/packager.sh
+++ b/node_modules/react-native/scripts/packager.sh
@@ -8,17 +8,12 @@
 THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
 REACT_NATIVE_ROOT="$THIS_DIR/.."
 # Application root directory - General use case: react-native is a dependency
-PROJECT_ROOT="$THIS_DIR/../../.."
+PROJECT_ROOT="$THIS_DIR/../../../apps/demo-react-native"
 
 # check and assign NODE_BINARY env
 # shellcheck disable=SC1090
 source "${THIS_DIR}/node-binary.sh"
 
-# When running react-native tests, react-native doesn't live in node_modules but in the PROJECT_ROOT
-if [ ! -d "$PROJECT_ROOT/node_modules/react-native" ];
-then
-  PROJECT_ROOT="$THIS_DIR/.."
-fi
 # Start packager from PROJECT_ROOT
 cd "$PROJECT_ROOT" || exit
 "$NODE_BINARY" "$REACT_NATIVE_ROOT/cli.js" start "$@"
diff --git a/node_modules/react-native/scripts/react-native-xcode.sh b/node_modules/react-native/scripts/react-native-xcode.sh
index 3ef9a71..9e60516 100755
--- a/node_modules/react-native/scripts/react-native-xcode.sh
+++ b/node_modules/react-native/scripts/react-native-xcode.sh
@@ -60,7 +60,7 @@ esac
 REACT_NATIVE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 # The project should be located next to where react-native is installed
 # in node_modules.
-PROJECT_ROOT=${PROJECT_ROOT:-"$REACT_NATIVE_DIR/../.."}
+PROJECT_ROOT=${PROJECT_ROOT:-"$REACT_NATIVE_DIR/../../apps/demo-react-native"} 
 
 cd "$PROJECT_ROOT" || exit
 